### DISCUSSION TITLE
Zoom in the image using touch pad

### DISCUSSION THREAD

**Discussion by parsagholipour** [igordanchenko/yet-another-react-lightbox#181] [Q&A] [answered]
I'm using the library as a carousel. The user can swipe right and left using their touch pad but cannot zoom in. Is there any way to support the zooming function too? I mean the browser's zooming function not the library. I think the swipe right/left functionality overrides browser's. Is there a way to disable it?

**Comment by igordanchenko** [comment, marked as answer]
This is not something that is supported by the library. Disabling the features you mention above severely affects usability of touch-swipe navigation on mobile devices, and this is not something that I'm open to offer as an officially supported configuration option. You can see the drawbacks I'm talking about by adding the following CSS:

```css
.yarl__container, .yarl__slide_image {
    touch-action: unset;
}
```
